assets/app/view/game_card.rb

```diff
@@ -299,5 +299,14 @@ def render_broken
         ]),
       ])
     end
+
+    private
+
+    def game_card_round_info
+      game = Kernel.const_get(Engine::GAMES_BY_TITLE[@gdata['title']].to_s)
```

Comment:
isn't there a method in engine for this? why do you need to const_get

Reply:
It creates a symbol, which I can use to call the class method with.

Example:
class Test
  def self.fum; end
end
a = "Test"
b = Kernel.const_get(a)

a.fum throws exception
b.fum works

Well, I did not find any prettier solution, but maybe there is?